Utils/DrippyClient.ts

```diff
@@ -1,12 +1,47 @@
-import { Client, Collection, SlashCommandBuilder } from "discord.js";
+import { Client, Collection, ColorResolvable, EmbedBuilder, SlashCommandBuilder, TextChannel, WebhookClient } from "discord.js";
 import { Config } from "./Config";
-// Vibe added made custom properties on the Client which works well... Until I transfered this project to typescript and now no types so this should fix it.
+import { DrippyBlockClient } from "./DrippyBlockClient";
 
+// Vibe added made custom properties on the Client which works well... Until I transfered this project to typescript and now no types so this should fix it.
 export class DrippyClient extends Client {
     config: Config;
     events: Collection<string, (...args) => Function>;
     commands: Collection<string, Command>;
     cooldowns: string[];
+    bot: DrippyBlockClient;
+    webhook: WebhookClient;
+
+    sendWebHookMessage(message: string, picture: string = "https://yt3.ggpht.com/ytc/AMLnZu8gDqmPezdXMDI1k183oQeknA_V4ZDb6FQPo39PVg=s88-c-k-c0x00ffffff-no-rj", username?: string) {
```

Comment:
Are we sure this is a permanent url?

Reply:
probably better to make this a config option?

Reply:
I’m probably going to go down that route